src/components/global/componentWrapper.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import PropTypes from 'prop-types';
+import { Child } from './componentWrapper.styles';
+import GlobalStyle from '../globalStyle';
+import defaultTheme from '../themes';
+
+export const GlobalAndThemeWrapper = (props) => {
+  const { theme, children, isBold } = props;
+  if (children === undefined) {
+    return null;
+  }
+
+  return (
+    <ThemeProvider theme={theme}>
+      <>
+        <GlobalStyle />
```

Comment:
It's better to have a prop to check if use GlobalStyle is necessary or not. then decide to render it. For example, as I told before, the Avatar component doesn't need GlobalStyle.

Reply:
By the way, it's better to move this component to a specific folder like utils and rename with a more meaningful name.